javascript-create-module/index.js

```diff
@@ -1,153 +1,96 @@
 #!/usr/bin/env node
-
-// Usage: npx @jahia/create-module@latest module-name [namespace]
-
 import fs from "node:fs";
 import path from "node:path";
+import process from "node:process";
 import { fileURLToPath } from "node:url";
-import { replaceInFileSync } from "replace-in-file";
 import camelCase from "camelcase";
+import { styleText } from "node:util";
 
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = path.dirname(__filename);
+try {
+  console.log(styleText("bgBlueBright", " Jahia JavaScript Module Creator "), "\n");
 
-// Show help if no argument is provided
-const [, , projectName, moduleType = "module", namespace = camelCase(projectName || "")] =
-  process.argv;
+  const nodeVersion = Number(process.versions.node.split(".")[0]);
 
-if (!projectName) {
-  console.log(`\x1B[1m## Usage\x1B[0m
+  if (nodeVersion < 22) {
+    console.warn(
+      `You are using ${styleText("redBright", `Node.js ${process.versions.node}`)} which is not officially supported.
+Please upgrade to ${styleText("greenBright", "Node.js 22 or later")} if you encounter any issues.
+Upgrade guide: ${styleText("underline", "https://nodejs.org/en/download")}
+`,
+    );
+  }
 
-    npx @jahia/create-module@latest project-name [module-type] [namespace-definitions]
+  const name = process.argv[2];
 
-where
-- \x1B[1mproject-name\x1B[0m (mandatory) can be anything you want to call your project
-- \x1B[1mmodule-type\x1B[0m (optional) Can be one of:
-  - \x1B[3mtemplatesSet\x1B[0m A collection of templates and components. A template set is required when creating a website.
-  - \x1B[3mmodule\x1B[0m sStandard Jahia module. This is the default value.
-  - \x1B[3msystem\x1B[0m Critical module for the whole platform .
-- \x1B[1mnamespace-definitions\x1B[0m (optional) The namespace used for content definitions. Default is the project name in camel case.
-`);
-  process.exit(9);
-}
+  if (!name) {
+    console.warn(
```

Comment:
```suggestion
    console.error(
```

Reply:
Fun thing: there's not difference between these two functions in nodejs, they just write to stderr. I'll update it though